libioc/Firewall.py

```diff
@@ -124,6 +134,18 @@ def _offset_rule_number(
         _rule_number = int(rule_number)
         return str(_rule_number + self.IPFW_RULE_OFFSET)
 
+    def _exec_list(
+        self,
+        command: typing.List[str],
+        ignore_error: bool=False
+    ) -> str:
+        try:
+            return str(libioc.helpers.exec(command, ignore_error=ignore_error))
```

Comment:
Please apply this change directly to the `_exec` method and change its return type to `str`. It will not interfere with other methods using _exec, as they will all ignore the "returned" `None`.